tests/test_start_date.py

```diff
@@ -23,6 +25,27 @@ class StartDateTest(BaseTapTest):
       is greater than or equal to the start date
     """
 
+    def get_properties(self, original: bool = True):
+        return_value = {
+            'start_date': '2021-04-01T00:00:00Z',
+            'shop': 'talenddatawearhouse',
+            'date_window_size': 30,
+            # BUG: https://jira.talendforge.org/browse/TDL-13180
```

Comment:
Can you throw this in base as well, please. Or what I usually do is mark it as `BUG_<n> : jira-link` in one place and then mark everywhere in the tests with `BUG_<n>` that it applies.

Reply:
Can do